src/libostree/ostree-sepolicy.c

```diff
@@ -257,6 +260,18 @@ initable_init (GInitable     *initable,
         freecon (con);
       }
 
+      {
+        g_autofree guchar *csum_raw = NULL;
+        if (!ostree_checksum_file (etc_selinux_dir, OSTREE_OBJECT_TYPE_FILE,
```

Comment:
Mmm...I would say use `ot_checksum_file` instead of this.  We don't actually care about uid/gid/perms or xattrs on the file I'd say...or more realistically the kernel and userspace selinux policy tools I'm almost sure don't.

As a bonus that function already returns an ASCII checksum.